CHANGELOG.md

```diff
@@ -2,17 +2,19 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
+## StreamChat
+### 🐞 Fixed
+- Saving payloads to local database is now 50% faster. Initial launch and displaying channel list should be noticeably faster [#1973](https://github.com/GetStream/stream-chat-swift/issues/1973)
+- Fix not waiting for last batch of events to be processed when connecting as another user [#2016](https://github.com/GetStream/stream-chat-swift/issues/2016)
+
 ## StreamChatUI
 ### ✅ Added
 - Add Support for Slow Mode [#1953](https://github.com/GetStream/stream-chat-swift/pull/1953)
+- Present channel screen modally when channel list in not embedded by navigation controller [#2011](https://github.com/GetStream/stream-chat-swift/pull/2011)
+- Show channel screen as right detail when channel list is embedded by split view controller [#2011](https://github.com/GetStream/stream-chat-swift/pull/2011)
```

Comment:
I'm a bit concerned about this. `SplitViewController` has additional logic. Can we test these scenarios currently in the DemoApp? If not, how do we know they are working?

Reply:
I tested the demo on all size classes including iPhones Pro/Pro max transitioning from portrait (cr) to landscape (rc) when split appears. Also tried scenarios without split/navigation. I can add manual tests but I don't think it makes much sense for DemoApp